ExperimentManager1.py

```diff
@@ -48,14 +48,14 @@ def __init__(self):
         self.punishment_time = 0.004
         self.center_reward_time = 1
 
-        # Initialize CSV file and video capture
+        # Initialize CSV file and video capture       # Micky: I think it is time to delete these lines. You moved them to the relevant classes
         # self.initialize_csv_logging()
         # self.initialize_video_capture()
 
         ##
         ### VARIABLES FOR LOG FILE INITIALLY INITIALIZED TO FALSE
-        self.data_file_loc = 'path_to_csv_file.csv'
-        self.cc_var = False
+        self.data_file_loc = 'path_to_csv_file.csv'   # Micky: Use relative file name. it will make the SW more portable
```

Comment:
yes

Reply:
I am keeping the comments until I see the change